docker-compose.yml

```diff
@@ -279,12 +279,13 @@ services:
 
   media-proxy:
     image: gennyproject/media-proxy:latest
+    platform: linux/amd64
     container_name: media-proxy
     env_file:
       - ${ENV_FILE}
     environment:
       # - KEYCLOAKURL=https://keycloak-office.gada.io/auth/realms
-      - KEYCLOAKURL=https://keycloak.gada.io/auth:${GENNY_KEYCLOAK_URL}/realms
+      - KEYCLOAKURL=https://keycloak-bali.gada.io/realms
```

Comment:
instead of hard code keycloak url here, I suggest you use ${GENNY_KEYCLOAK_URL} like you already did 
`KEYCLOAKURL=${GENNY_KEYCLOAK_URL}/realms` is more flexible

Reply:
Fixed.